Source/Android/app/src/main/res/values/arrays.xml

```diff
@@ -101,8 +101,8 @@
     <string-array name="serialPort1DeviceEntries" translatable="false">
         <item>Nothing</item>
         <item>Dummy</item>
-        <item>Broadband Adapter (XLink Kai)</item>
-        <item>Broadband Adapter (Built In)</item>
+        <item>Broadband Adapter (HLE)</item>
+        <item>Broadband Adapter (LLE XLink Kai)</item>
```

Comment:
You can't reorder entries either.

Reply:
Okay.  Redoing this pull request my bad.

Reply:
Should be resolved now.